data-loader/core/build.gradle

```
@@ -41,3 +44,67 @@ spotbugsTest.reports {
     html.enabled = true
 }
 spotbugsTest.excludeFilter = file("${project.rootDir}/gradle/spotbugs-exclude.xml")
+
+if (project.gradle.startParameter.taskNames.any { it.endsWith('publish') } ||
+        project.gradle.startParameter.taskNames.any { it.endsWith('publishToMavenLocal') }) {
+    // not to publish the fat jar to maven central
+    shadowJar.enabled = false
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            artifactId = 'scalardb-data-loader'
+            from components.java
+            pom {
+                name = 'ScalarDB Data Loader'
+                description = 'A tool for importing data to and from ScalarDB'
```

Maybe `exporting` is necessary? (CC: @ypeckstadt, @josh-wong)
```suggestion
                description = 'A tool for exporting data from and importing data into ScalarDB'
```